test/agent-policy-tests/test-script-utils.sh

```diff
@@ -0,0 +1,395 @@
+#!/usr/bin/env bash
```

Comment:
You might want to use bats for testing: https://github.com/bats-core/bats-core

Reply:
Yup, switched to using bats, added it to `task_helper_functions.sh`, and made a new target for running these tests in the Makefile.